footer-component.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Footer Component.
+ *
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.component('brFooter', {
+    controller: Ctrl,
+    templateUrl: requirejs.toUrl(
+      'bedrock-angular-footer/footer-component.html'),
+  });
+}
+
+/* @ngInject */
+function Ctrl($rootScope, config) {
+  var self = this;
+  self.route = $rootScope.route;
```

Comment:
If it works and has been tested against multiple routes where the navbar gets switched off/on when clicking links to switch routes -- then I guess we're good. Would be good to understand the details. I guess the `$rootScope.route` object changes internally.